### ISSUE TITLE
The received tab is wrongly opened in immersive mode after exiting sleep mode

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [MozillaReality/FirefoxReality#2244]
**[Notes]:**
- This behavior also occurs in normal view. After exiting sleep mode, the received tab will automatically be opened in the focused window.
- As a side effect of this issue, the video continues to play in the background, and it's still playing after exiting immersive mode, until the video is paused from the tab that was initially played.

**[Affected Versions]:**
- Firefox Reality version 6 RC5 (33a031c7)

**[Affected Platforms]:**
- Oculus Go - Android 7.1.2
- Oculus Quest - Android 7.1.1
- HTC Vive Focus Plus - Android 7.1.2
- HTC Vive Focus - Android 7.1.2

**[Prerequisites]:**
- The Firefox Reality browser is opened and the Settings menu is displayed.
- Have an valid FxA account.
- Make sure that the FxA account is logged on the VR device and also on a second device(PC, mobile..)

**[Steps to reproduce]:**
1. Open a 360 video and play it in immersive mode.
2. Send a tab from your PC to the VR device.
5. While in immersive mode take the headset off for ~ 15 seconds.
6. Put the headset back on and observe the behavior.

**[Expected results]:**
- The video opened continues to be played in immersive mode.

**[Actual results]:**
- The received tab is automatically opened in immersive mode.

**[Additional Notes]:**
- The "New Tab Opened" notification is also displayed.
- Attached is a screen recording of the issue: [link](https://drive.google.com/open?id=1WtTh7fGXWhV6ju_fsun0WWMPPCJjfbMr).

**Comment by Softvision-RemusDranca** [comment]
After further investigation, it seems that this issue also affects normal videos that are playing before receiving the tab. The received tab is automatically opened after exiting sleep mode and the video from the previous tab continues to be played in the background and can only be paused from the previous tab.

**Comment by Softvision-RemusDranca** [comment]
I have verified this issue and it's no longer reproducible with the latest Firefox Reality version 7 RC1 (b772e6d7) build. Now, the received tab is not opened in immersive/theatre mode after exiting sleep mode.
However, the "New Tab Added" notification is not longer displayed after exiting immersive mode, this behavior is documented in issue: #2283. 

Tested on the Oculus Quest (Android 7.1.1), Oculus Go (Android 7.1.2), HTC Vive Focus Plus (Android 7.1.2) and HTC Vive Focus (Android 7.1.2) devices.
